src/storage/heap_file.c

```diff
@@ -14507,6 +14507,22 @@ heap_dump (THREAD_ENTRY * thread_p, FILE * fp, HFID * hfid, bool dump_records)
   fprintf (fp, "\n\n*** END OF DUMP FOR HEAP FILE ***\n\n");
 }
 
+/*
+ * heap_dump_specific_file () - dump a specific heap file with class name
+ *
+ * return            :
+ * thread_p (in)     : thread entry
+ * fp (in)           : output file
+ * dump_records (in) : true to dump records
+ * class_name (in)   : name of class to dump
+ */
+void
+heap_dump_specific_file (THREAD_ENTRY * thread_p, FILE * fp, bool dump_records, const char *class_name)
+{
+  /* TODO: Fetch HFID of class which corresponds to class_name, and dump heap file by HFID. Will be handled at CBRD-25313 and CBRD-25314. */
+  fprintf (fp, "\n\n*** DUMP A CLASS NAMED %s ***\n\n", class_name);
```

Comment:
Does this need to print out this message? This information is already included in the heading.

Reply:
I think it's duplicated information. It's been deleted.